CMakeModules/FindLIBVLC.cmake

```diff
@@ -6,19 +6,16 @@ find_path(LIBVLC_INCLUDE_DIR vlc/vlc.h
     HINTS
         ${PC_LIBVLC_INCLUDEDIR}
         ${PC_LIBVLC_INCLUDE_DIRS}
+        /usr/local/opt/vlc/include
```

Comment:
Because we're using the official vlc binaries now and because that ships a bunch of libraries that are shipped by homebrew otherwise, I made it a keg_only formula